src/lambda/get-blog/utils.py

```diff
@@ -0,0 +1,103 @@
+import functools
+import boto3
+import json
+import logging
+from decimal import Decimal
+from urllib import parse
+
+dynamodb = boto3.resource('dynamodb')
+table    = dynamodb.Table('blogs')
+
+class DecimalEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return float(obj)
+        return json.JSONEncoder.default(self, obj)
+
+class JsonPayloadBuilder:
+    payload = {}
+
+    def add_status(self, success):
+        self.payload['success'] = success
+        return self
+
+    def add_data(self, data):
+        self.payload['data'] = data
+        return self
+
+    def add_message(self, msg):
+        self.payload['message'] = msg
+        return self
+
+    def compile(self):
+        return json.dumps(self.payload, cls=DecimalEncoder, ensure_ascii=False).encode('utf8')
+
+
+def api_response(code, body):
+    return {
+        "isBase64Encoded": False,
+        "statusCode": code,
+        'headers': {
+            "Access-Control-Allow-Origin": '*',
+            "Content-Type": "application/json",
+            "Referrer-Policy": "origin"
+        },
+        "multiValueHeaders": {"Access-Control-Allow-Methods": ["POST", "OPTIONS", "GET", "PATCH", "DELETE"]},
```

Comment:
Currently we don't allow methods other than GET and Options on feeds endpoint

Reply:
Fixed.